streampark-flink/streampark-flink-kubernetes/src/main/scala/org/apache/streampark/flink/kubernetes/FlinkTrackController.scala

```diff
@@ -236,3 +240,19 @@ object MetricCache {
   def build(): MetricCache = new MetricCache()
 
 }
+
+class ArchivesCache {
+  def put(k: String, v: String): Unit = cache.put(k, v)
+
+  def get(k: String): String = cache.getIfPresent(k)
+
+  def asMap(): Map[String, String] = cache.asMap().toMap
+
+  def cleanUp(): Unit = cache.cleanUp()
+
+  val cache: Cache[String, String] = Caffeine.newBuilder.build()
```

Comment:
The cache size or expire policy are not set, why use cache here?

Reply:
The FlinkJobStatusWatcher.dowatch() method needs to get the archived path corresponding to the task when scheduling. The archived path is put into the cache when the job is started, and the path in the cache is retrieved when the job is scheduled

Reply:
I mean the cache size or expire policy are not set, why don't you use Map directly?

Reply:
The underlying Caffeine is map. i am using the logic of the previous cache here. Cache expiration I added this: 
def invalidate(key: ClusterKey): Unit = cache.invalidate(key) 
to control.

Reply:
Map supports `put`, `get` and `invalidate`, why use Cache?